### ISSUE TITLE
Add "tags" to ProductFragment

### ISSUE THREAD

**Issue by kashifimran** [Shopify/js-buy-sdk#765]
**Please complete the checklist before filing an issue:**

### Feature details 

It almost seems like an omission that product tags are not included in the ProductFragment query. 

**Describe the solution you'd like**
Any chance we'll get "tags" array in ProductFragment?

**Comment by arobbins** [comment]
I think the reason they keep tags excluded from the `ProductFragment` is because the more fields you add the slower the resulting graph query becomes.

As a workaround (what I'm doing), you can use the unoptimized version of the library and then add the tags yourself.

**Comment by aksperiod** [comment]
I agree that tags should be baked in. I think that custom queries should only be needed with more complicated use-cases, such as accessing 3rd-party product metadata, however right now you have to use a custom query for the pretty default case of `tags`.

**Comment by jeffdaley** [comment]
+1

Seems reasonable to expect that information entered in the inventory UI would be available by default.

**Comment by Spence-S** [comment]
was super annoyed at this so I published this here. Everything is exactly the same as exact tags is now in the product fragment https://github.com/Spence-S/js-buy-sdk

**Comment by Nonemoticoner** [comment]
How possibly this is still not supported? 😑

**Comment by Hagafny** [comment]
Guys, are you planning to add support for tags? It seems like with the new version there's no way to get them at all

**Comment by Hagafny** [comment]
> > was super annoyed at this so I published this here. Everything is exactly the same excepts that tags is now in the product fragment
> > https://www.npmjs.com/package/shopify-buy-with-tags
> > `yarn add shopify-buy-with-tags`
> > Will republish to keep up to date with this repo.
> 
> This was utterly amazing for my application. However it seems to use a very outdated version of the Shopify API.

Yes, I was using this also but I had to use the unoptimized version and hack a few queries myself to get the most updated version of the Shopify API

**Comment by teodormalmborg** [comment]
> 

Would you mind shareing how you made 

> > > was super annoyed at this so I published this here. Everything is exactly the same excepts that tags is now in the product fragment
> > > https://www.npmjs.com/package/shopify-buy-with-tags
> > > `yarn add shopify-buy-with-tags`
> > > Will republish to keep up to date with this repo.
> > 
> > 
> > This was utterly amazing for my application. However it seems to use a very outdated version of the Shopify API.
> 
> Yes, I was using this also but I had to use the unoptimized version and hack a few queries myself to get the most updated version of the Shopify API

Would you mind sharing how you made it work?

I moved nuxt-shopify into my local modules and hacked it to use shopify-buy-with-tags instead of buy-with-tags. However,  when I change apiVersion 2022-01 to 2022-10 nothing seems to work and the error stack is impossible to decrypt for me.

    const config = {
        domain: '<%= options.domain %>',
        storefrontAccessToken: '<%= options.storefrontAccessToken %>',
        language: '<%= options.language %>',
        apiVersion: '2022-01'

    };

**Comment by skeddles** [comment]
```
async function getTags (productId) {
	const productsQuery = client.graphQLClient.query((root) => {
		root.addConnection('products', { args: { first: 250 } }, (product) => {
			product.add('title');
			product.add('tags');
		});
	});
	let result = await client.graphQLClient.send(productsQuery);
	return result.data.products.edges.find(p => p.node.id === productId).node.tags;
}
```
(requires the unoptimized build)

**Comment by AliRehman7141** [comment]
There should be a options to add tags
